en/guides/manage-devices/index.md

```diff
@@ -0,0 +1,137 @@
+
+```
+{
+  "type": "model",
+  "authority-id": "THE-SSO-ACCOUNT-ID",
+  "brand-id": "THE-SSO-ACCOUNT-ID",
+  "system-user-authority": [
+    "ANOTHER-SSO-ACCOUNT-ID",
+    "YET-ANOTHER-SSO-ACCOUNT_ID"
+  ],
+  "series": "16",
+  "model": "mymodel",
+  "architecture": "armhf",
+  "kernel": "pi2-kernel",
+  "gadget": "cm3",
+  "timestamp": "2017-07-11T15:55:59+00:00"
+}
+```
+
+In this case, any one who can log in to either ANOTHER-SSO-ACCOUNT-ID or YET-ANOTHER-SSO-ACCOUNT_ID accounts can sign system-user assertions with keys registered to those accounts. And such system-user assertions are valid for systems built with this model.
+
+### Self-signed system-user assertions
+
+A model can also specify that anyone with a registered key can create a valid system-user assertion. This is done with an asterisk ("*") in the `system-user-authority` field:
+
+```
+{
+  "type": "model",
+  "authority-id": "THE-SSO-ACCOUNT-ID",
+  "brand-id": "THE-SSO-ACCOUNT-ID",
+  "system-user-authority": "*",
+  "series": "16",
+  "model": "mymodel",
+  "architecture": "armhf",
+  "kernel": "pi2-kernel",
+  "gadget": "cm3",
+  "timestamp": "2017-07-11T15:55:59+00:00"
+}
+```
+
+## Creating a system-user assertion
+
+We now know that creating a valid system-user assertion is limited by the model assertion. And, there is information in the model assertion that is needed to create the system-user assertion, specifically the `brand-id` and the `model`.
+
+
+There's a snap that makes this part easy: make-system-user
+
+1. Install it 
+
+        $ snap install make-system-user --classic
+
+1. Check its help:
+
+        $ make-system-user.run --help
+
+    The help specifies that you need the brand, which is the `brand-id` field's value in the model assertion. 
+
+    You also need the value of the `model` field, in this case: `mymodel`
+
+    You need to specify a username and password. 
+
+    And, you need to name the key to sign the assertion. Note that the assertion only works on a given system if the key is within the limitations of the model assertion as described previously.
+
+
+1. Here is a sample execution matching the above:
+
+```code
+$ sudo make-system-user.run --brand THE-SSO-ACCOUNT-ID --model mymodel --username chuckthecoolcat --password heresapassword --key MYVALIDKEY
+You need a passphrase to unlock the secret key for
+user: "MYVALIDKEY"
+4096-bit RSA key, ID C375E301, created 2016-01-01
+
+Done. You may copy auto-import.assert to a USB drive and insert it into an unmanaged Core system, after which you can log in using the username and password you provided.
+```
+
+The result is `auto-import.assert` in your current directory. This can be used on an qualifying image (described above) that is not already *managed* to make a system user. 
+
+Simply copy `auto-import.assert` to the root directory of a USB drive, insert it, and the system user is created. 
```

Comment:
two lines above it says "insert it into an unmanaged Core system", so I think it is already clear.